pkg/client/shub/pull.go

```diff
@@ -50,7 +50,7 @@ func DownloadImage(filePath string, shubRef string, force, noHTTPS bool) (err er
 	// Get the image manifest
 	manifest, err := getManifest(ShubURI, noHTTPS)
 	if err != nil {
-		return fmt.Errorf("Failed to get manifest from Shub: %v", err)
+		return fmt.Errorf("failed to get manifest from Shub: %v", err)
```

Comment:
this is correct, but has nothing to do with the described change

Reply:
Same story here, I'll revert this file.